dashboard/src/actions/repos.test.tsx

```diff
@@ -493,32 +493,15 @@ describe("installRepo", () => {
     expect(store.getActions()).toEqual(expectedActions);
   });
 
-  it("uses kubeapps own namespace if namespace is _all", async () => {
-    await store.dispatch(
-      repoActions.installRepo("my-repo", "_all", "http://foo.bar", "", "", "", []),
-    );
-
-    expect(AppRepository.create).toHaveBeenCalledWith(
-      "default",
-      "my-repo",
-      "kubeapps-namespace",
-      "http://foo.bar",
-      "",
-      "",
-      {},
-      [],
-    );
-  });
-
```

Comment:
Yes, you are assuming right. I think the toggle will be explanatory enough: "**Show** repos in all namespaces". The problem is that the repos are indeed more confusing because we have: single namespace, global namespace and all namespaces. Making that distinction is difficult (I wonder if we should have gone for the `ClusterAppRepository` CRD option vs global repositories)